src/ServicePulse.Host/app/js/directives/ui.particular.multi-checkboxlist.js

```diff
@@ -0,0 +1,104 @@
+﻿
+; (function (window, angular, undefined) {
+    'use strict';
+
+
+    function controller($scope) {
+
+        var selectActions = Object.freeze({
+            Selection: 1,
+            Deselection: 2
+        });
+
+        $scope.selectRow = function(row, value){
```

Comment:
Doesn't look like this is used anywhere in the view, so we could drop this from scope